camel/societies/role_playing.py

```diff
@@ -256,12 +272,22 @@ def init_agents(self, init_assistant_sys_msg: BaseMessage,
             output_language (str, optional): The language to be output by the
                 agents.
         """
+        if self.assistant_functions is not None:
+            assistant_config = FuncConfig(
+                functions=[
+                    func.as_dict() for func in self.assistant_functions
+                ], function_call="auto")
```

Comment:
Should we create a class method called `from_openai_function_list ` in `FuncConfig` to make it easier?

Like:
```
@classmethod
def from_openai_function_list(cls, function_list: List[OpenAIFunction], function_call="auto"):
    return cls(functions=[
                    func.as_dict() for func in function_list
                ], function_call=function_call)
```
So we can do:
```
assistant_config = FuncConfig.from_openai_function_list(self.from_openai_function_list)
```

Reply:
I think this implementation is the best as it does not break the current settings